src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,23 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+ul,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  list-style: none;
```

Comment:
This rule is for `ul`, so then you will need to use this rule again for example here 
```
.nav {
  margin: 0;
  padding: 0;
  display: flex;
  list-style: none;
}
```